_posts/2020-03-19-quarkus-1-3-0-final-released.adoc

```diff
@@ -0,0 +1,206 @@
+---
+layout: post
+title: 'Quarkus 1.3.0.Final released - New class loader infrastructure, GraalVM 20 support, and a lot more'
+date: 2020-03-19
+tags: release
+synopsis: Quarkus 1.3.0.Final comes with a new class loader infrastructure, GraalVM 20 support, 8 new extensions and a lot of bugfixes and improvements.
+author: gsmet
+---
+
+It has been a bit less than two months since our last major Quarkus release
```

Comment:
Is the intention here to explain the long gap between releases?

Reply:
No, I'm just factual.

Reply:
I am a little on the fence... I don't really like it, but if I were adding it, I would probably would have started with `In case you haven't noticed, ....`